### DISCUSSION TITLE
Slash Commands Not Loading Whenever Bot Uploaded to Hosting Server

### DISCUSSION THREAD

**Discussion by AkDProjects** [discord-jda/JDA#2400] [Questions and Help]
Hello, I have an issue that I'm really confused about.

I have a bot that has slash commands, whenever I run that bot from my local machine and add it to a guild, the slash commands load and I'm able to use them.
When I upload that bot to a server(hosting site), the slash commands don't load to any new guild I add the bot too.
BUT, the slash commands are still present on any old guilds I had the bot in before hosting it, and the slash commands still work, I even get the input from the slash commands in the server logs.

Would anyone have a guess why? I can provide code and logs if needed. 
Thank you.

**Comment by HagarJNode** [comment]
Hmm... this sounds a bit of what I see after updating to 5.0.0 beta. Running my bot locally adds the slash command to one of the guilds on the Discord, but not all of them, and uploading the bot doesn't yield anything, as I see it so fare.
